lib/LedgerNodePeers.js

```diff
@@ -0,0 +1,147 @@
+/*!
+ * Copyright (c) 2017-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const bedrock = require('bedrock');
+const database = require('bedrock-mongodb');
+const {util: {BedrockError}} = bedrock;
+const {COLLLECTION_NAME} = require('./ledgerNodePeerCollection');
+
+/**
+ * The LedgerNodePeers class exposes peer information.
+ */
+module.exports = class LedgerNodePeers {
+  constructor({ledgerNode}) {
+    this.ledgerNode = ledgerNode;
+    this.ledgerNodeId = ledgerNode.id;
```

Comment:
I would say that the type of validation you are describing is valuable for APIs that developers interact with on a regular basis.  This constructor is, and probably will only ever be used in *exactly* one spot in our code base.  As long as we've ensured that that one usage works as expected (which we have) we should be good.  

A similar use case is private helper APIs that we use in places.  Since they are not public facing, we don't necessarily need to add a bunch of validation.

The additional validation steps don't cost us a lot of compute, but they are also not free either.